### PR TITLE
Added links to tech stack

### LINKED ISSUE #173: Update README.md
I would like to work on the README.md file of this repo and make some minor changes like
+ Restructuring content for a better appeal
+ Adding badges for tech tack and discord for a cool effect

I would like to be assigned to the issue if approved!

### PR DESCRIPTION
**Pull request by the-amazing-atharva**
+ Added links to tech stack
+ Added target="_blank" attribute so that it opens in new tab

Thoughts @lugenx ?

Fixes #173

### CODE REVIEW

**Review by lugenx — CHANGES_REQUESTED** [review]
thank you, this looks good when rendered, but, Is there a way to do it without using HTML? I want to keep it markdown only for now for simplicity.

**Comment by the-amazing-atharva** [comment]
Will have to research a bit, working on it
Will update here!

**Comment by the-amazing-atharva** [comment]
Hey!
I converted it to markdown but it seems that we can only open links in the same tab

**Comment by tsimian** [comment]
> Hey! I converted it to markdown but it seems that we can only open links in the same tab

Yeah, I don't think this is currently possible with plain markdown. Perhaps we could still use an anchor tag, but only for each tech like so...
```
## Tech Stack

- Front End: <a href="https://reactjs.org/" target="_blank">React (JavaScript)</a>, <a href="https://mui.com/" target="_blank">Material UI</a>
```

What are your thoughts @lugenx?

**Comment by tsimian** [comment]
> @the-amazing-atharva thank you, i just have one comment about replacing the MongoDB link. @tsimian, i still think we should go with markdown only, do you think opening on the same page is an issue? wouldn't most people click with the cmd/ctrl for this kind of link anyway? 🤔

Yeah, I agree. There are other methods to open in a new tab. Though I felt it was worth mentioning that plain markdown does not support the target attribute. Linking to the documentation is a good idea though.

**Comment by the-amazing-atharva** [comment]
Thanks @tsimian and @lugenx 
Do I have to change anything?? 
Something regarding the MongoDB link was mentioned above ig

**Review by lugenx — CHANGES_REQUESTED** [review]
sorry looks like my comment related to mongo db stayed pending, I haven't posted it, you should see it now.

File: README.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ It will visualize users' habits by tracking their activities (recycling, commuti
...
 
+
 ## Tech Stack
 
-- Front End: React (JavaScript), Material UI
-- Backend: NodeJS/ExpressJS (JavaScript)
-- Database: MongoDB
+*   **Front End:** [React](https://reactjs.org/), [Material UI](https://mui.com/) (JavaScript)
+*   **Backend:** [NodeJS](https://nodejs.org/en/docs)/[ExpressJS](https://expressjs.com/) (JavaScript)
+*   **Database:** [MongoDB](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwjbkKiXmOL9AhUGmWYCHTVNBA0YABAAGgJzbQ&ohost=www.google.com&cid=CAASJuRoPXQHwswO-Z51bFJxJzJQgk5pfhUHekUpyg4nhr0O165dFo0C&sig=AOD64_0as2Ii1qr1y7WdpN8D6ldm4-qj8A&q&adurl&ved=2ahUKEwi566KXmOL9AhVTTHwKHaNBC0kQ0Qx6BAgIEAE)
```
[the remark this answers is not in the document]
Comment (lugenx) [reply README.md:31]: @the-amazing-atharva, any update on this?

[the remark this answers is not in the document]
Comment (the-amazing-atharva) [reply README.md:31]: On it brother!! @lugenx

**Comment by the-amazing-atharva** [comment]
Updated the MongDB section
Any more changes @lugenx ?

**Comment by lugenx** [comment]
looks good @the-amazing-atharva , thank you 👍

**Review by lugenx — APPROVED** [review]

**Comment by the-amazing-atharva** [comment]
> looks good @the-amazing-atharva , thank you 👍

It was a pleasure working with you @lugenx 😄

**Pull request merged**